src/features/Students/StudentsTable/columns.jsx

```diff
@@ -34,6 +34,16 @@ const getColumns = props => [
     accessor: 'created',
     Cell: ({ row }) => new Date(row.values.created).toUTCString(),
   },
+  {
+    Header: 'First Access',
+    accessor: 'first_access',
+    Cell: ({ row }) => new Date(row.values.first_access).toUTCString(),
+  },
+  {
+    Header: 'Last Access',
+    accessor: 'last_access',
+    Cell: ({ row }) => new Date(row.values.last_access).toUTCString(),
+  },
```

Comment:
Also it's necessary add the grade column.

Reply:
@anfbermudezme that's true.